.github/workflows/main.yml

```diff
@@ -16,7 +16,10 @@ jobs:
       with:
         go-version: '1.17' # The Go version to download (if necessary) and use.
     - run: sudo apt install -y universal-ctags pandoc
-    - run: go test ./...
+    - run: mkdir .tools && wget https://github.com/daedaleanai/llvm-project/releases/download/ddln-llvm14-rc/llvm-14.0.0.tar.gz && tar -xf llvm-14.0.0.tar.gz -C .tools
```

Comment:
Generally speaking, third-party users would use llvm-14 from upstream, no need for daedaleans. However, for our usecase it makes sense to use this one, because it differs a bit from upstream llvm and we have some fixes, so in order to properly parse our code, we should use this version.

Regarding the setup_workspace,  I'm not quite sure how that works, but I think the idea was to integrate reqtraq with dbt as a dependency, where we could specify the pinned llvm versions we have and where they are (given llvm is also a dbt dependency).

Reply:
I added a build tag to conditionally add support for libclang. This way, whoever doesn't need libclang, does not need to install llvm (e.g.: prod repos). I think that  resolves this question.